python/setup.py

```diff
@@ -220,6 +220,7 @@ def get_packages(self):
             "opencensus",
             "prometheus_client >= 0.7.1, < 0.14.0",
             "smart_open",
+            "pydantic",
```

Comment:
remove it since we don't do migration in this PR?

Reply:
I am using pydantic in this PR.